mlrun/model_monitoring/model_monitoring_batch.py

```diff
@@ -811,12 +812,19 @@ def run(self):
                     "hellinger_mean": drift_result["hellinger_mean"],
                 }
 
-                self.frames.write(
-                    backend="tsdb",
-                    table=self.tsdb_path,
-                    dfs=pd.DataFrame.from_dict([tsdb_drift_measures]),
-                    index_cols=["timestamp", "endpoint_id", "record_type"],
-                )
+                try:
+                    self.frames.write(
+                        backend="tsdb",
+                        table=self.tsdb_path,
+                        dfs=pd.DataFrame.from_dict([tsdb_drift_measures]),
+                        index_cols=["timestamp", "endpoint_id", "record_type"],
+                    )
+                except v3io_frames.errors.Error:
+                    logger.warn(
+                        "No TSDB schema file found at the target path, probably due to not enough model events",
```

Comment:
First add the "what" and then you can say "why"
```suggestion
                        "Events were not written to the TSDB: No TSDB schema file found at the target path, probably due to not enough model events",
```

Reply:
Right, fixed :)